CHANGELOG.md

```diff
@@ -534,3 +531,13 @@ Released with 1.0.0-beta.37 code base.
 ### Fixed
 -  Fix jsonrpc payload and response types (#4743) (#4761)
 -  Allowed more flexibility in typing the overly constrained `provider.disconnect` function (#4833)
+
+## [1.7.3]
```

Comment:
@ankushnodereal  Why there are version bumps, build files and package lock updates in this PR? We dnt include these changes from external PRs, and we include such changes from internal team members during release only.